worker/backend/process_killer_test.go

```diff
@@ -3,10 +3,10 @@ package backend_test
 import (
 	"context"
 	"errors"
-	"fmt"
 	"math"
 	"syscall"
 	"time"
+	"fmt"
```

Comment:
Not sure why these tests are [failing to compile now](https://ci.concourse-ci.org/teams/main/pipelines/prs/jobs/unit/builds/899#L5e839dab:535:540), but it's claiming `fmt` is not used. Maybe run `goimports` on all the files? (I noticed some formatting issues in several files)

Reply:
d01e3fd2c39929c5510c6ab2e6f43058ffe9f3eb